index.bs

```diff
@@ -432,12 +439,18 @@ Report Long Animation Frames {#loaf-processing-model}
 
     1. [=list/Append=] the [=duration=] between |safeTaskStartTime| and |safeTaskEndTime| to |timingInfo|'s [=task durations=].
 
-    1. If the user agent believes that updating the rendering of |document|'s [=node navigable=] would have no visible effect, then [=report frame timing=] given |document| and return.
+    1. If the user agent believes that updating the rendering of |document|'s [=node navigable=] would have no visible effect, then:
+        1. set |document|'s [=nearest same-origin root=]'s [=current frame timing info=] to null.
+
+        1. Let |frameDuration| be the [=duration=] between the [=relative high resolution time=] given |timingInfo|'s [=frame timing info/start time=] and |global|,
+            and the [=relative high resolution time=] given |unsafeTaskEndTime| and |global|.
+
+        1. If |frameDuration| is equal or greater than 50 milliseconds, then [=queue a long animation frame entry=] given |document|, |timingInfo|, and a new [/=paint timing info=] whose [=rendering update end time=] is 0.
```

Comment:
I think that presentation time would default to null, so perhaps you don't explicitly need to set it to 0.

But for an implementation which typically reports presentation times I do think we should report 0 and not leave the option undefined.  Does this need to be explicit?

Reply:
Done